website/server/controllers/api-v3/auth.js

```diff
@@ -101,21 +101,25 @@ api.registerLocal = {
     let existingUser = res.locals.user; // If adding local auth to social user
 
     req.checkBody({
-      email: {
-        notEmpty: {errorMessage: res.t('missingEmail')},
-        isEmail: {errorMessage: res.t('notAnEmail')},
-      },
       username: {
-        notEmpty: {errorMessage: res.t('missingUsername')},
```

Comment:
why was this line changed?

Reply:
Before, when a field was left empty, the error was something like "Invalid email address" or "Login Name must be 1 to 20 characters." Now, with `notEmpty` set to true, it says "Missing email" or "Missing Login Name" as intended.